test/parallel/test-net-socket-timeout.js

```diff
@@ -67,7 +67,7 @@ for (let i = 0; i < invalidCallbacks.length; i++) {
   );
 }
 
-const server = net.Server();
+const server = net.Server((conn) => { /* do nothing */ });
```

Comment:
is `mustNotCall` better?

```suggestion
const server = net.Server(common.mustNotCall());
```

Reply:
@Himself65 the callback is called sometime. Anyway the `conn` argument is not used so it could be removed.

Reply:
FTR, I added the `conn` parameter to convey that we're receiving a connection object but are explicitly ignoring it.

Reply:
So said, add the equal check is easier for understand?

Reply:
Okay, done.